PY to PHP

LoveMyData opened the pull request:
current scraper adding 5 records but deleting 5 records at the same time.

henare commented on scraper.php:
```diff
@@ -0,0 +1,102 @@
+        'address'           => $address,
+        'description'       => preg_replace('/\s+/', ' ', trim(html_entity_decode($record->find('span', 1)->plaintext))),
+        'info_url'          => $term_url,
+        'comment_url'       => $comment_base . trim($record->find('a',0)->plaintext),
```
I think this results in an invalid URL because it's not escaped.

henare commented on scraper.php:
```diff
@@ -0,0 +1,102 @@
+    
+    $address   = preg_replace('/\s+/', ' ', trim(html_entity_decode($record->find('div', 2)->plaintext)));
+    $address   = rtrim($address, ".");
+    $address   = $address . ", Australia";
```
Since the PlanningAlerts geocoder biases for Australia we don't need this.

henare commented on scraper.php:
```diff
@@ -0,0 +1,102 @@
+    echo $record->find('div')->plaintext;
+    
+    $address   = preg_replace('/\s+/', ' ', trim(html_entity_decode($record->find('div', 2)->plaintext)));
+    $address   = rtrim($address, ".");
```
We're getting an extra full stop at the end of the address, is this the culprit? e.g. `101 Hawthorne Avenue, CHATSWOOD WEST NSW 2067., Australia`

henare commented on scraperwiki.php:
```diff
@@ -0,0 +1,159 @@
+<?php
```
It would be much better if this and the other library files weren't included in the repository and was installed by composer like other scrapers. You can see how to make this change here: https://morph.io/documentation/scraperwiki

henare commented:
@LoveMyData nice :snowflake: I've added a few comments, once those are resolved we're good to merge.

LoveMyData commented:
@henare, the dot was removed with earlier commits, we need to seriously show the data in descending order. I will remove the Australia later tonight, tks

henare commented:
Just clear your database before doing another run that fixes something and you should be right :+1:

LoveMyData commented:
Removed 'Australia' from address
removed error msg
however, can't use composer, there is very lack of doc on morph.io web site to make it work. I did try....

henare commented:
Nice, thanks @LoveMyData :+1:
